Makefile

```diff
@@ -128,7 +128,7 @@ vet: gowork ## Run go vet against code.
 	go vet ./api/...
 
 .PHONY: tidy
-tidy: fmt
+tidy: ## Run go mod tidy on every mod file in the repo
 	go mod tidy
```

Comment:
I believe the latest pre-commmit (`.pre-commit-config.yaml`) intended for us to run the `fmt` target.

So we probably need to update it and change line `args: ["tidy"]` to something `args: ["tidy", "fmt"]` right?

Reply:
yeah, that makes sense

Reply:
As far as I see pre-commit runs golangci-lint https://github.com/openstack-k8s-operators/cinder-operator/blob/082c9624f769a83dba2bdef9a3ab4e53bf7f776f/.pre-commit-config.yaml#L1-L6 
And that is configured to run fmt: https://github.com/openstack-k8s-operators/cinder-operator/blob/082c9624f769a83dba2bdef9a3ab4e53bf7f776f/.golangci.yaml#L7